.env.sample

```diff
@@ -1,2 +1,9 @@
 MINISCOPE_IO_BASE_DIR="~/.config/miniscope_io"
-MINISCOPE_IO_LOGS__LEVEL="INFO"
\ No newline at end of file
+MINISCOPE_IO_LOGS__LEVEL="INFO"
+MINISCOPE_IO_CSVWRITER_BUFFER=1000
+MINISCOPE_IO_SERIAL_BUFFER=100
+MINISCOPE_IO_FRAME_BUFFER=100
+MINISCOPE_IO_IMAGE_BUFFER=100
+MINISCOPE_IO_STREAM_HEADER_PLOT_HISTORY=500
+MINISCOPE_IO_STREAM_HEADER_PLOT_UPDATE_MS=100
+MINISCOPE_IO_STREAM_HEADER_PLOT_KEY="timestamp,linked_list,frame_buffer_count"
```

Comment:
gonna move these to submodels bc we're getting a little splayed out here

Reply:
nice